gitlab/map.jinja

```diff
@@ -40,3 +40,16 @@ Debian:
   - python-gitlab
 {%- endload %}
 {% set client = salt['grains.filter_by'](client_defaults, merge=salt['pillar.get']('gitlab:client')) %}
+{% set client = salt['grains.filter_by']({
```

Comment:
Why? I would keep installing python-gitlab package as a default as it seems to be working fine now.